odoo/cli/help.py

```diff
@@ -6,17 +7,17 @@
 class Help(Command):
     """ Display the list of available commands """
 
-    template = """\
-usage: {prog_name} [--addons-path=PATH,...] <command> [...]
+    template = textwrap.dedent("""
```

Comment:
dedent doesn't strip
```suggestion
    template = textwrap.dedent("""\
```

Reply:
... doesn't strip? 😉 😉 

![immagine](https://github.com/user-attachments/assets/73b81053-66cd-48df-8c17-12f33bcf172e)

Ok, it's terrible and inappropriate.
Thanks for the advice. 👍🏻